packages/strapi-admin/admin/src/translations/es.json

```diff
@@ -0,0 +1,163 @@
+  "app.components.HomePage.createBlock.content.second": "\u0020le ayudará a definir la estructura de datos de sus modelos. Si eres nuevo aquí, te recomendamos encarecidamente que sigas nuestros\u0020",
+  "app.components.HomePage.createBlock.content.tutorial": "\u0020tutorial.",
+  "app.components.HomePage.button.quickStart": "INICIAR EL TUTORIAL DE INICIO RÁPIDO",
+  "app.components.HomePage.button.blog": "VER MÁS EN EL BLOG",
+  "app.components.HomePage.support": "APOYANOS",
+  "app.components.HomePage.support.content": "¡Al comprar la camiseta, nos permitirá continuar nuestro trabajo en el proyecto para darle la mejor experiencia posible!",
+  "app.components.HomePage.support.link": "CONSIGUE TU CAMISETA AHORA",
+
+  "app.components.BlockLink.documentation": "Lea la documentación",
+  "app.components.BlockLink.documentation.content": "Descubra los conceptos, guías de referencia y tutoriales.",
+  "app.components.BlockLink.code": "Ejemplos de código",
+  "app.components.BlockLink.code.content": "Aprenda probando proyectos reales desarrollados por la comunidad.",
+
+
+  "app.components.InputFile.newFile": "Añadir nuevo archivo",
+  "app.components.InputFileDetails.open": "Abrir en una nueva pestaña",
+  "app.components.InputFileDetails.remove": "Eliminar este archivo",
+  "app.components.InputFileDetails.originalName": "Nombre original:",
+  "app.components.InputFileDetails.size": "Tamaño:",
+
+  "app.components.ImgPreview.hint": "Arrastre y suelte el archivo en esta área o {browse} para cargar un archivo.",
+  "app.components.ImgPreview.hint.browse": "buscar",
+
+  "app.components.InstallPluginPage.helmet": "Mercado - Plugins",
+  "app.components.InstallPluginPage.title": "Mercado - Plugins",
+  "app.components.InstallPluginPage.description": "Extienda su aplicación sin esfuerzo.",
+  "app.components.InstallPluginPage.plugin.support-us.description": "¡Apóyanos comprando la camiseta Strapi. Eso nos permitirá seguir trabajando en el proyecto y tratar de darle la mejor experiencia posible!",
+  "app.components.InstallPluginPage.InputSearch.label": " ",
+  "app.components.InstallPluginPage.InputSearch.placeholder": "Buscar un plugin... (ej: autenticación)",
+  "app.components.InstallPluginPopup.downloads": "descargar",
+  "app.components.InstallPluginPopup.navLink.description": "Descripción",
+  "app.components.InstallPluginPopup.navLink.screenshots": "Capturas de pantalla",
+  "app.components.InstallPluginPopup.navLink.avis": "opinión",
+  "app.components.InstallPluginPopup.navLink.faq": "preguntas frecuentes",
+  "app.components.InstallPluginPopup.navLink.changelog": "changelog",
+  "app.components.InstallPluginPopup.noDescription": "No hay descripción disponible",
+
+  "app.components.LeftMenuFooter.poweredBy": "Potenciado por ",
+  "app.components.LeftMenuLinkContainer.configuration": "Configuraciones",
+  "app.components.LeftMenuLinkContainer.general": "General",
+  "app.components.LeftMenuLinkContainer.installNewPlugin": "Plaza de mercado",
```

Comment:
Not sure about this 🤔 ... I don't know where is used, but without see it I think is better just "Mercado"